frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx

```diff
@@ -161,7 +161,11 @@ const PipelineRunTableRow: React.FC<PipelineRunTableRowProps> = ({
       {customCells}
       {hasRowActions && (
         <Td isActionCell dataLabel="Kebab">
-          <ActionsColumn items={actions} />
+          <ActionsColumn
+            data-testid="pipeline-run-table-row-actions"
+            items={actions}
+            popperProps={{ appendTo: () => document.body, position: 'right' }}
```

Comment:
if you append to the body it gives different text styling compared to you appending it to the dashboard root

Reply:
Updated, thanks.